crates/chisel/bin/main.rs

```diff
@@ -46,6 +48,10 @@ pub struct ChiselParser {
     #[command(subcommand)]
     pub sub: Option<ChiselParserSub>,
 
+    /// Path to a Solidity file to import as a prelude
+    #[clap(long, help_heading = "REPL options")]
+    pub prelude: Option<PathBuf>,
```

Comment:
should this be a Vec instead? so multiple files can be loaded?

maybe even a dir?

Reply:
added a dir and some spiffy log msgs